.scala-steward.conf

```diff
@@ -0,0 +1,7 @@
+pullRequests.frequency = "@weekly"
```

Comment:
This can be monthly. We are not updating and releasing Twirl in anger and that all dependencies must be update.

Reply:
Done